### ISSUE TITLE
Add a CONTRIBUTING file

### ISSUE THREAD

**Issue by cfhammill** [Mouse-Imaging-Centre/RMINC#102]
Gabe has recently asked how to get involved in RMINC development. It may be useful to add a CONTRIBUTING file that explains the package design philosophy and how to contribute effectively.

**Comment by jasonlerch** [comment]
Good idea. Though really the core part is pull requests/patches - welcome any time!
